lib/invoker.rb

```diff
@@ -112,6 +120,13 @@ def check_and_notify_with_terminal_notifier(message)
       end
     end
 
+    def notify_with_libnotify(message)
```

Comment:
I am in favour of printing this warning when Invoker starts. Bit like - https://github.com/code-mancers/invoker/blob/master/lib/invoker/cli.rb#L112

This way user still knows how to get additional functionality and stdout will not be too noisy. As such, if libnotify is not installed, we will be printing warning each time when we want to notify.